src/meta-client/src/client/procedure.rs

```diff
@@ -199,6 +199,7 @@ impl Inner {
                             times += 1;
                             continue;
                         } else {
+                            warn!("An error occurred in gRPC, status: {status}");
```

Comment:
The same log is also needed here.: https://github.com/fengjiachun/greptimedb/blob/main/src/meta-client/src/client/cluster.rs#L240